ActindoWorkFlow/UserInterface/04a_ProcessActions.md

```diff
@@ -1,104 +1,71 @@
-
-The list displays all actions that are matching the selected status. Depending on the settings, the displayed columns may vary. All fields are read-only. Click an action to display the corresponding process in the *Process ID* view, see [Process ID](./03a_Processes.md#process-id).
-
-> [Info] When clicking an action in the *Process actions* view, the selected action is displayed in the *Actions* tab of the corresponding *Process ID* view. The *Actions* tab of the *Process ID* view is, in fact, an excerpt of the *Process actions* menu entry. Click the *Logs* tab in the *Process ID* view for detailed information about the selected process actions, see [Logs](./06a_Logs.md).
+The following functions are available for the editing toolbar. Depending on the process status, the editing toolbar displays specific buttons:
 
 - [x]    
     Select the checkbox to display the editing toolbar. If you click the checkbox in the header, all actions in the list are selected.
 
-Depending on the action status, the editing toolbar displays the following buttons:
-
 - ![Retry](../../Assets/Icons/Retry01.png "[Retry]") (Retry)  
-    Click this button to retry the action. This button is displayed if the action status is *Error*.
+    Click this button to retry the action. This button is displayed if the action status is *Error*, *Done*, or *Aborted*.
+
+- ![Abort](../../Assets/Icons/Stop.png "[Retry]") (Abort)    
+    Click this button to abort the action. This button is displayed if the action status is *Error* or *In progress*.
 
 - ![Suspend](../../Assets/Icons/Pause.png "[Suspend]") (Suspend)  
-    Click this button to suspend the action temporarily. This button is displayed if the action status is *In progress*. When clicking this button, the action status changes to *Suspended*.
+    Click this button to temporarily suspend the action, for example, if you want to evaluate a process in detail and want to pause the process execution. This button is displayed if the action status is *In progress*<!---Status "In progress" gibt es nicht mehr. Status Active?-->. When clicking this button, the action status changes to *Suspended*.
 
 - ![Start](../../Assets/Icons/Play.png "[Start]") (Start)  
     Click this button to resume the action. This button is displayed if the action status is *Suspended*. When clicking this button, the action status changes again to *In progress*. The warning sign in the *Suspended actions* column of the *Processes* view is no longer displayed.
 
 - [REMOVE DEFER TIME]  
-    Click this button to skip the defer time preconfigured in the action, if any. The action is then executed immediately. This button is displayed if the action status is *In progress*.
+    Click this button to skip the defer time preconfigured in the action, if any. The action is then executed immediately. This button is displayed if the action status is *In progress*<!---Status "In progress" gibt es nicht mehr. Status Active?-->.
+
```

Comment:
Es kann gut sein, dass es jetzt Active heißt, also einfach konsistent umbenennen? In progress wird für Processes verwendet. Evtl. mit Entwicklung klären, ob es so heißen soll, aber für mich klingt plausibel.